### ISSUE TITLE
Duplicate data field issue on show hide columns and table rendering

### ISSUE THREAD

**Issue by vickyrathee** [wenzhixin/bootstrap-table#2085]
I've one table which is using AJAX data to load and working fine, but i need to use one field multiple times in table to make links, with check-box and more to associate the IDs.
```
 <tr>
        <th data-field="id" data-checkbox="true"></th>
        <th data-field="id" data-sortable="true">Item ID</th>
        <th data-field="id" data-formatter="linkFormatter" data-sortable="true">Item Link</th>
    </tr>
```
But when i use the same field the heading style don't synced properly and the show/hide columns is also broken.

See the screenshot and this JsFiddle :: http://jsfiddle.net/4r6g4cfu/194/

![image](https://cloud.githubusercontent.com/assets/6106479/13850198/fc1c7a92-ec7e-11e5-8c34-e708c2b08938.png)

Looking for help

**Comment by vickyrathee** [comment]
Adding screenshot as well for rendering problem with hiding the first column form show/hide.

![image](https://cloud.githubusercontent.com/assets/6106479/13850198/fc1c7a92-ec7e-11e5-8c34-e708c2b08938.png)

**Comment by vickyrathee** [comment]
@wenzhixin Looking for a comment please.

**Comment by djhvscf** [comment]
Let me get a try with this

**Comment by djhvscf** [comment]
This issue is fixed in #2106 

See this example: http://jsfiddle.net/djhvscf/4r6g4cfu/219/

**Comment by djhvscf** [comment]
Close this issue and open a new one with a new issue related to this one
